Mathlib/Analysis/Normed/Field/WithAbs.lean

```diff
@@ -16,7 +17,7 @@ on an absolute value. This is useful when dealing with several absolute values o
 In particular this allows us to define the completion of a field at a given absolute value.
 -/
 
-open Topology
+open Filter Topology
```

Comment:
`Filter` seems to be needed only for one result, you might either `open in` before the declaration, or have a `Filter.` in the proof, no?